### PR TITLE
Updating dependencies

### PR DESCRIPTION
**Pull request by andrewrlee**
Removing unused dependency

Speeding up tests - running tests in band reduces duration by 1/3rd - 1/2th 

### Context

https://trello.com/c/Pol58w06/

### Intent

Update to latest version of dependencies

### Checklist

- [ ] This PR contains **only** changes related to the above card
- [ ] Tests have been added/updated to cover the change
- [ ] Documentation has been updated where appropriate
- [ ] Tested in Development

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
-  "parser": "babel-eslint",
+  "parser": "@babel/eslint-parser",
```
Comment (andrewrlee) [inline .eslintrc:2]: related to babel-eslint being [very out of date](https://stackoverflow.com/a/69557309)

File: .prettierrc [hunk 2]
```diff
@@ -3,8 +3,6 @@
   "bracketSpacing": true,
   "htmlWhitespaceSensitivity": "css",
   "insertPragma": false,
-  "jsxBracketSameLine": false,
-  "jsxSingleQuote": false,
```
Comment (andrewrlee) [inline .prettierrc:7]: no longer valid options in latest parser (and irrelevant for us)

File: package.json [hunk 3]
```diff
@@ -11,7 +11,7 @@
     "css-build": "./bin/build-css",
     "clean": "rm -rf public/* .port.tmp *.log build/* uploads/* test-results.xml",
     "lint": "eslint . --cache --max-warnings 0",
-    "test": "jest",
+    "test": "jest --runInBand",
```
Comment (andrewrlee) [inline package.json:14]: running in a single thread is a lot faster than trying to run in parallel - between 1/3rd and a half faster

File: server/services/cms.js [hunk 4]
```diff
@@ -32,11 +32,8 @@ const {
 class CmsService {
   #cmsApi;
 
-  #contentRepository;
```
Comment (andrewrlee) [inline server/services/cms.js:35]: removed this as it wasn't being used

**Review by leonk — APPROVED** [review]
LGTM

**Review by davidthomas22 — COMMENTED** [review]
LGTM

**Pull request merged**